### PR TITLE
make the kdump initrd before the FS becomes read only (master)

### PR DESCRIPTION
**Pull request by mvidner**

### CODE REVIEW

**Comment by mvidner** [comment]
This is a merge of #73 from CaaSP to master

File: src/modules/Kdump.rb [hunk 1]
```diff
@@ -385,10 +385,24 @@ def ReadKdumpSettings
     #
     # @return [Boolean] whether successful
     def update_initrd
+      # For CaaSP we need an explicit initrd rebuild before the
+      # first boot, when the root filesystem becomes read only.
+      rebuild_cmd = "/usr/sbin/tu-rebuild-kdump-initrd"
+      # part of transactional-update.rpm
+      update_initrd_with("if test -x #{rebuild_cmd}; then #{rebuild_cmd}; fi")
```
Comment (jreidinger) [inline src/modules/Kdump.rb]: in fact PBI said, that it should be enabled only if kdump is enabled, but I do not see in code that happen. Do you test that this is not called when kdump is not enabled?

Comment (mvidner) [inline src/modules/Kdump.rb]: For the installation, the code returns early enough.
For a regular SLES running system, transactional-update.rpm will not be present.

Comment (jreidinger) [inline src/modules/Kdump.rb]: so maybe add it to comment, so it is clear enough

**Pull request merged**